### ISSUE TITLE
tensorflow docs have sigmoid_cross_entropy_with_logits args flipped

### ISSUE THREAD

**Issue by cpdurham** [tensorflow/tensorflow#6700]
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/functions_and_classes/shard5/tf.nn.sigmoid_cross_entropy_with_logits.md 

labels comes before logits, please excuse if i'm way off base, not sure where these docs are generated from

**Comment by yaroslavvb** [comment]
Arg order change was introduced in https://github.com/tensorflow/tensorflow/commit/333dc32ff79af21484695157f3d141dc776f7c02 by @martinwicke ... I guess docs need to be regenerated?

**Comment by cpdurham** [comment]
wow, isn't this going to break upgraded code in very hard to debug ways?

**Comment by yaroslavvb** [comment]
This is one of a number of "hard to debug breaking changes" coming up in preparation to 1.0 release, documented [here](https://github.com/tensorflow/tensorflow/blob/c72d891418c326794a913cc13382c104255fe1f2/RELEASE.md#breaking-changes-to-the-api) . The recommendation for backward compatibility is to not use any positional arguments but specify keywords explicitly

**Comment by martinwicke** [comment]
And in fact, the new interface disallows positional arguments for that
exact reason.

**Comment by ppwwyyxx** [comment]
Some other function, e.g. sparse_softmax_cross_entropy_with_logits also have this breaking change. These need to be documented in RELEASE.md.

But in one function: [tf.nn.weighted_cross_entropy_with_logits](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/functions_and_classes/shard8/tf.nn.weighted_cross_entropy_with_logits.md) , positional arguments are not disallowed. This may lead to silent failure.

**Comment by martinwicke** [comment]
That's a good point. We should enforce names args for that function as
well.

**Comment by cpdurham** [comment]
argument order was addressed in 1.0 with scripts for compatibility to add named fields (for google searches)
